std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
> Any suggestions on how to approach the issue and find out why the tester is not complaining?

You need to find why your test is broken. You might be missing a check which ensures that everything has gone as expected. If you need more time for this, I suggest we revert the PR for now.